client/webserver/webserver_test.go

```diff
@@ -347,11 +348,21 @@ func TestLoadMarket(t *testing.T) {
 	link := newLink()
 	s, tCore, shutdown, _ := newTServer(t, false)
 	defer shutdown()
-	_, err := link.cl.Connect(tCtx)
+	linkWg, err := link.cl.Connect(tCtx)
 	if err != nil {
 		t.Fatalf("WSLink Start: %v", err)
 	}
-	defer link.cl.Disconnect()
+
+	// This test is not running WebServer or calling handleWS/websocketHandler,
+	// so manually stop the marketSyncer started by wsLoadMarket and the WSLink
+	// before returning from this test.
+	defer func() {
+		link.cl.feedLoop.Stop()
+		link.cl.feedLoop.WaitForShutdown()
```

Comment:
The `feedLoop` (`marketSyncer`) uses a package level logger that gets written on `New` server construction in subsequent test.  Gotta wait for it to be done.